Translations are identified by position and these two new messages
were inserted out of order so non-english users get a segv.

Patch by Martin von Gagern.

http://bugs.gentoo.org/97541

--- man/msgs/mess.en
+++ man/msgs/mess.en
@@ -21,10 +21,6 @@
 	"No manual entry for %s\n"
 PAGER_IS
 	"\nusing %s as pager\n"
-BROWSER_IS
-	"\nusing %s as browser\n"
-HTMLPAGER_IS
-	"\nusing %s to dump HTML pages as text"
 SYSTEM_FAILED
 	"Error executing formatting or display command.\n\
 System command %s exited with status %d.\n"
@@ -172,6 +168,10 @@
 	"error: no TROFF command specified in %s\n"
 NO_CAT_FOR_NONSTD_LL
 	"no cat page stored because of nonstandard line length\n"
+BROWSER_IS
+	"\nusing %s as browser\n"
+HTMLPAGER_IS
+	"\nusing %s to dump HTML pages as text"
 FOUND_FILE
 	"manfile_from_sec_and_dir() found %s\n"
 CALLTRACE1
